tests/solids/LiH_solid_1x1x1_pp/md_rc_vmc_LiH-gamma.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<simulation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.mcc.uiuc.edu/qmc/schema/molecu.xsd">
+  <project id="md_rc_vmc_LiH-gamma" series="1">
+    <application name="qmcapp" role="molecu" class="serial" version="0.2">
+      Simple Example of solid LiH
+    </application>
+  </project>
+
+  <!--random seed="666999"/-->
+  <qmcsystem>
+  <simulationcell name="global">
+    <parameter name="lattice" units="bohr">
+   -3.550000000000000    0.000000000000000    3.550000000000000
+    0.000000000000000    3.550000000000000    3.550000000000000
+   -3.550000000000000    3.550000000000000    0.000000000000000
+    </parameter>
+    <parameter name="bconds">
+p p p
+    </parameter>
+    <parameter name="LR_dim_cutoff">
+15
+    </parameter>
+  </simulationcell>
+  </qmcsystem>
+   <particleset name="i" size="2">
+    <group name="H">
+      <parameter name="charge">       1.0     </parameter>
+      <parameter name="atomicnumber"> 1.0     </parameter>
+    </group>
+    <group name="Li">
+      <parameter name="charge">       3.0     </parameter>
+      <parameter name="atomicnumber"> 3.0     </parameter>
+    </group>
+    <attrib name="ionid" datatype="stringArray">
+Li
+H
+    </attrib>
+    <attrib name="position" datatype="posArray" condition="1">
+    0.000000000000000    0.000000000000000    0.000000000000000
+    0.500000000000000    0.500000000000000    0.500000000000000
+    </attrib>
+  </particleset>
+  <particleset name="e" random="yes" random_source="i">
+    <group name="u" size="2">
+      <parameter name="charge">
+          -1
+      </parameter>
+    </group>
+    <group name="d" size="2">
+      <parameter name="charge">
+          -1
+      </parameter>
+    </group>
+  </particleset>
+  <wavefunction name="psi0" target="e">
+    <determinantset type="einspline" href="LiH-gamma.pwscf.h5" tilematrix="1 0 0 0 1 0 0 0 1" twistnum="0" gpu="yes" meshfactor="1.0" source="i"  precision="single">
+      <comment text="create the spline-based basis set builder" />
+      <basisset name="spline-basis" type="einspline" />
+
+      <comment text="create the spline-based single particle orbital sets" />
+      <sposet name="spo-spline-fixed" basisset="spline-basis" size="6" optimizable="no" />
+      <sposet name="spo-up"    basisset="spo-spline-fixed" size="6" optimizable="no" />
+      <sposet name="spo-dn"    basisset="spo-spline-fixed" size="6" optimizable="no" />
+
```

Comment:
@chrislzhao before you start, please pull the most recent change.
Could you please remove all the basisset. I understand that you need that for optimization but we will not go through the AoS routine but only the SoA route via LCAOSet which already has rotation support. For this reason, how to setup the basisset input will be different. In this PR, we need to minimal input that supports running MSD solid.